src/tools/clippy/tests/ui/custom_ice_message.stderr

```diff
@@ -9,3 +9,5 @@ note: we would appreciate a bug report: https://github.com/rust-lang/rust-clippy
 
 note: Clippy version: foo
 
+query stack during panic:
+we're just showing a limited slice of the query stack
```

Comment:
I think it shouldn't say this if all parts of the query stack have been printed. Even when the maximum length is limited.

Reply:
@bjorn3 so I guess I should normalize it right?
and also when I run the test, it fails, and it deletes the const-pat-ice.stderr file
do you think that we should remove it?